src/core/manager.c

```diff
@@ -3904,18 +3904,17 @@ static void manager_send_ready_user_scope(Manager *m) {
         m->status_ready = false;
 }
 
-static void manager_send_ready_system_scope(Manager *m) {
+static void manager_send_ready_finished(Manager *m) {
         int r;
 
         assert(m);
 
-        if (!MANAGER_IS_SYSTEM(m))
-                return;
-
         /* Skip the notification if nothing changed. */
         if (m->ready_sent && m->status_ready)
                 return;
 
+        /* Note that for user managers, we might have already sent READY=1 in manager_send_ready_user_scope().
+         * But we still need to flush STATUS=. */
```

Comment:
maybe say explicitly that the 2nd READY=1 doesn't hurt and is a NOP for the hosting service manager.